angular/src/shared/core.less

```diff
@@ -244,4 +244,9 @@ topbar-languageswitch {
 
 .action-button {
     top: -9px;
-}
\ No newline at end of file
+}
+
+// force all material design form fields to be 100% wide
```

Comment:
I'm a huge fan of this.  To date each control has done it for themselves, but I strongly feel this should be the default behavior.  I will localize it if you like though.